src/feedback/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Specific feedback functions:
+# Main class holding and running feedback functions:
+import importlib.metadata
+
+from trulens.feedback.embeddings import Embeddings
```

Comment:
Did the additions to `src/feedback` get copied by mistake? The source should be in `src/feedback/trulens/feedback`